torch/_dynamo/utils.py

```diff
@@ -275,12 +275,6 @@ def setup_compile_debug():
     compile_debug = os.environ.get("TORCH_COMPILE_DEBUG", "0") == "1"
 
     if compile_debug:
-        torch._logging.set_logs(
```

Comment:
did you mean to check these in under this PR?

Reply:
yeah